src/importsource.cpp

```diff
@@ -14,12 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include "libcellml/importsource.h"
```

Comment:
This shouldn't have been removed (this is how we do things now: add the `.cpp`'s corresponding `.h` file first, then system headers then API headers, with an empty line between each category), and an empty line added.

Reply:
81328612